### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by accurateKevin**
- Clarify whitelisting process in Unity
- Add links to website documentation
- Language edits

### CODE REVIEW

**Comment by accurateKevin** [comment]
@andreaKin please review.

**Comment by andreaKin** [comment]
@Ronserruya  and @prime31  Kevin made a number of edits to the Unity intro document based on feedback we received from developers.

Please review, edit and merge as appropriate.

My recommendation is that you then take this and turn it into a new section in the official docs called "Unity". It should be placed after the iOS section and before the Python section so that we have all client SDKs first and all back-end SDKs after.

File: README.md
Comment (Ronserruya) [inline README.md:93]: The method name was correct, in c# methods names are in CamelCase, unlike java where its "lowerCamelCase"

File: README.md
Comment (Ronserruya) [inline README.md:62]: Why was the section about "storeKey" removed?

File: README.md
Comment (Ronserruya) [inline README.md:88]: > In the production environment Kin clients are not authorized to create new accounts  

Clients cant do it on testnet as well.

And its not the clients are not authorized, the functionality is just not exposed on the client sdk

**Comment by Ronserruya** [comment]
Added some comments, lets wait for @prime31 to review as well before merging

**Review by Ronserruya — CHANGES_REQUESTED** [review]
Added some comments, lets wait for @prime31 to review as well before merging

**Review by prime31 — APPROVED** [review]

**Pull request closed without merging**
